src/libs/utils.js

```diff
@@ -625,11 +625,30 @@ export const getColumnSort = (getList, callback) => {
   };
 };
 
+const sortVisibleTable = ({ list, sort }) => {
```

Comment:
List isn't defined when `NewResearcherConsole` first mounts, so this method ends up triggering error notifications on that page on mount. Easy way to fix this is to give it a default on the object restructuring statement. `{list = [], sort}` should be enough to fix it.